engines/ultima/ultima8/graphics/shape_frame.cpp

```diff
@@ -23,128 +23,60 @@
 #include "ultima/ultima8/misc/pent_include.h"
 
 #include "ultima/ultima8/graphics/shape_frame.h"
-#include "ultima/ultima8/convert/convert_shape.h"
-#include "ultima/ultima8/convert/u8/convert_shape_u8.h"
-#include "ultima/ultima8/filesys/idata_source.h"
+#include "ultima/ultima8/graphics/raw_shape_frame.h"
 
 namespace Ultima {
 namespace Ultima8 {
 
-/*
-  parse data and fill class
- */
-ShapeFrame::ShapeFrame(const uint8 *data, uint32 size, const ConvertShapeFormat *format,
-                       const uint8 special[256], ConvertShapeFrame *prev) : _line_offsets(0) {
-	// Load it as u8
-	if (!format || format == &U8ShapeFormat || format == &U82DShapeFormat)
-		LoadU8Format(data, size);
-	else if (format == &PentagramShapeFormat)
-		LoadPentagramFormat(data, size);
-	else if (format == &U8CMPShapeFormat)
-		LoadU8CMPFormat(data, size, format, special, prev);
-	else
-		LoadGenericFormat(data, size, format);
-}
+ShapeFrame::ShapeFrame(const RawShapeFrame *rawframe)
+: _xoff(rawframe->_xoff), _yoff(rawframe->_yoff),
```

Comment:
Missing indentation on the continuation strings.

Reply:
Ok, this mistake was all me :)